requirements.txt

```diff
@@ -77,6 +76,7 @@ typing_extensions==4.0.1
 urllib3==1.26.8
 virtualenv==20.13.0
 webassets==2.0
+webdriver-manager==3.7.0
```

Comment:
It would be good if test dependencies could be moved to a separate requirements file, as this increases our build/attack surface area by bundling all this stuff in our production app (out of scope for this PR though!)

Reply:
great idea - I will do this as part of a BAU for this repo! :)

Reply:
Check out my BAU PR :)